<launch>
  <node pkg="necst" name="Status" type="ROS_status.py"/>
  <node pkg="necst" name="tracking" type="ROS_tracking.py"/>
  <node pkg="necst" name="controller_client" type="ROS_controller.py"/>
  <node pkg="necst" name="antenna_server" type="ROS_antenna.py"/>
  <node pkg="necst" name="antenna_move" type="ROS_antenna_move.py"/>
  <node pkg="necst" name="encoder_status" type="ROS_encoder.py"/>
  <node pkg="necst" name="weather_status" type="ROS_weather.py"/>
  <node pkg="necst" name="abs_controller" type="ROS_abs.py"/>
  <node pkg="necst" name="m4_controller" type="ROS_m4.py"/>
  <node pkg="necst" name="authority_change" type="ROS_authority_check.py"/>
  <node pkg="necst" name="drive" type="ROS_drive.py"/>
  <node pkg="necst" name="dome" type="ROS_dome.py"/>
  <node pkg="necst" name="dome_tracking" type="ROS_dome_track_check.py"/>
</launch>
